recipes/plaster_pastedeploy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "plaster_pastedeploy" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c231130cb86ae414084008fe1d1797db7e61dc5eaafb5e755de21387c27c6fae
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pastedeploy >=1.5.0
```

Comment:
You probably only need `pip`, `python` and `setuptools` here.

Reply:
Only pip python and setuptools it is, also removed setuptools from run.

Reply:
Done.